processing/src/main/java/io/druid/segment/incremental/OffheapIncrementalIndex.java

```diff
@@ -92,6 +93,26 @@ public OffheapIncrementalIndex(
   }
 
   public OffheapIncrementalIndex(
```

Comment:
Yeah, I thought so too. Actually, I was considering creating Builders for `On/OffheapIncrementalIndex`, would that be better?